packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -269,6 +270,13 @@ export interface ApplicationLoadBalancedServiceBaseProps {
    * @default - false
    */
   readonly enableExecuteCommand?: boolean;
+
+  /**
+   * The load balancer idle timeout, in seconds
+   *
+   * @default 60
```

Comment:
We're not defaulting it to 60 seconds so I don't think this should say 60 here. I'd remove the `@default` altogether.

Reply:
Only issue with removing `@default` is that the linter and spec requires that optional properties must have `@default` documentation 

```error: [awslint:props-default-doc:@aws-cdk/aws-ecs-patterns.ApplicationLoadBalancedServiceBaseProps.idleTimeout] Optional property must have @default documentation ```

Reply:
What are your thoughts on explaining the behavior if nothing is provided
`@default - idle timeout is set to 60 seconds`

Reply:
Ah, forgot about that. Just put that it's undefined. I think it's fine to note that cloudformation will set it to 60 when undefined, but we're not setting anything on their behalf so let's make sure it doesn't look like the default is coming from our code.